### ISSUE TITLE
Admin Messages are always silent

### ISSUE THREAD

**Issue by CallMeSH** [sendbird/sendbird-sdk-flutter#16]
Hello!
We currently have a problem regarding the use of Admin Messages sent through the Platform API and the onChannelChanged callback of the Flutter SDK.

Even if silent is set to false manually on the Platform API (it is supposed to be the default value if we don’t send it but that doesn’t work either), the onChannelChanged callback isn’t triggered.
Whereas when we are inside a channel, we receive the onMessageReceived callback.
We have no problem with regular messages.

To sum up, it seems like Admin Messages are always considered silent for the onChannelChanged callback.
Could you help us with this issue ?

Thanks!

**Comment by intoxicated** [comment]
I will investigate and see if theres any problem

**Comment by intoxicated** [comment]
New patch will be released next week fyi
